l10n.js

```diff
@@ -176,6 +176,10 @@ document.webL10n = (function(window, document, undefined) {
 
     // handle escaped characters (backslashes) in a string
     function evalString(text) {
+      if (typeof text == 'undefined') {
+        return '';
+      }
+
```

Comment:
This was addressing an edge case, reaching the end of the file (however this should not be an issue anymore), and standard good-practice null pointer check?